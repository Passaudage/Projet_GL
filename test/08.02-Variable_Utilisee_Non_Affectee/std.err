Identifiants utilisés non initialisés :
   x : l.2 
