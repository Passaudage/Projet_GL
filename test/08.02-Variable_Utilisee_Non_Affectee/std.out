Analyse statique du code lutin :
